File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        int radius = 10;
        double xCenter = 0.0;
        double yCenter = 0.0;
        //"(x – a)² + (y – b)² = r²"
        double distancia =  Math.sqrt(Math.pow((xOfDart - xCenter),2) + Math.pow((yOfDart - yCenter), 2));
        if(distancia <= radius){
            if(distancia <= 1){
                return 10;
            }else if (distancia <= 5){
                return 5;
            }else {
                return 1;
            }
        }
        return 0;
    }
}

File: solutions/kotlin/darts/1/src/main/kotlin/Darts.kt
import kotlin.math.pow
import kotlin.math.sqrt

object Darts {

    fun score(x: Number, y: Number /* choose proper types! */): Int {
        val radius:Int = 10;
        val xCenter : Double = 0.0;
        val yCenter : Double = 0.0;
        //"(x – a)² + (y – b)² = r²"
        val distance: Double = sqrt(
                xCenter.minus(x.toDouble()).pow(2.0) + yCenter.minus(y.toDouble()).pow(2.0)
            );
        if(distance <= radius){
            return if(distance <= 1){
                10;
            }else if(distance <= 5){
                5;
            }else {
                1;
            }
        }
        return 0;
    }
}
